frontend/src/components/ComparisonEditors.tsx

```
@@ -383,34 +473,44 @@ const extractCodeAndExplanation = (text: string | null | undefined, language: st
-  // Extract only the code portion, ignore explanations
+  console.log("Generating preview for code:", code.substring(0, 100) + '...');
+  
+  // Extract just the code portion using the helper function
   const { code: extractedCode } = extractCodeAndExplanation(code, language);
-  console.log("Extracted code:", extractedCode);
+  
   switch (language.toLowerCase()) {
     case 'html':
       const isCompleteHTML = extractedCode.trim().toLowerCase().includes('<!doctype') || 
-                            extractedCode.trim().toLowerCase().includes('<html');
+                          extractedCode.trim().toLowerCase().includes('<html');
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)